modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/SleepTimer.kt

```diff
@@ -96,4 +133,16 @@ class SleepTimer @Inject constructor(@ApplicationContext private val context: Co
     private fun getAlarmManager(): AlarmManager {
         return context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
     }
+
+    private fun cleanUpSleepTimer() {
+        sleepTimeMs = null
+        cancelAutomaticSleepAfterTimeRestart()
+        cancelAutomaticSleepOnEpisodeEndRestart()
+    }
+    private fun cancelAutomaticSleepAfterTimeRestart() {
+        lastSleepAfterTime = null
+    }
+    private fun cancelAutomaticSleepOnEpisodeEndRestart() {
```

Comment:
Some empty space formatting is missing here as well.

Reply:
✅

I thought `./gradlew spotlessApply` would handle cases like this.

Reply:
Yeah, I'm surprised it doesn't complain about it. Maybe something to configure.